core/snippets/snippets_parser.py

```diff
@@ -5,6 +5,9 @@
 
 from .snippet_types import Snippet, SnippetVariable
 
+ESCAPED_SNIPPET_DELIMITER_EXPRESSION = re.compile(r"^\\---$", flags=re.MULTILINE)
+SNIPPET_DELIMITER = "---"
```

Comment:
Should probably move this one as well

https://github.com/talonhub/community/blob/65f614a41631996650a2bb5ff9988f13f838d03a/core/snippets/snippets_parser.py#L192